_pages/trl-reading-group.md

```diff
@@ -10,7 +10,7 @@ collection: books
 
 ## Description
 
-Amsterdam Lunch on Table (ALOT) is **reading group focussed on table representation learning and neural models for
+Amsterdam Lunch on Table (ALOT) is a **reading group focussed on neural models for
```

Comment:
OK but there's no reference to TRL indeed, which would seem good for people to understand, maybe going back to:

"Amsterdam Lunch on Table (ALOT) is a **reading group focused on table representation learning** and generally neural models for structured data."